src/test/java/com/iexec/sms/tee/config/TeeWorkerInternalConfigurationTests.java

```diff
@@ -29,52 +33,67 @@ class TeeWorkerInternalConfigurationTests {
     private static final String FINGERPRINT = "fingerprint";
     private static final String ENTRYPOINT = "entrypoint";
     private static final String LAS_IMAGE = "lasImage";
-    private static final long HEAP_SIZE_GB = 3;
+    private static final DataSize HEAP_SIZE = DataSize.parse("3GB");
     private static final long HEAP_SIZE_B = 3221225472L;
 
-    private final TeeAppProperties preComputeProperties = TeeAppProperties.builder()
-            .image("preComputeImage")
-            .fingerprint("preComputeFingerprint")
-            .entrypoint("preComputeEntrypoint")
-            .heapSizeInBytes(1L)
-            .build();
-    private final TeeAppProperties postComputeProperties = TeeAppProperties.builder()
-            .image("postComputeImage")
-            .fingerprint("postComputeFingerprint")
-            .entrypoint("postComputeEntrypoint")
-            .heapSizeInBytes(1L)
-            .build();
+    private TeeWorkerInternalConfiguration teeWorkerInternalConfiguration;
+    private TeeWorkerPipelineConfiguration pipelineConfig;
+    private TeeWorkerPipelineConfiguration.StageConfig validStageConfig;
```

Comment:
This field is only used in the `setUp` method. It can become a local variable as suggested by the IDE.